src/styles/blocks/header.scss

```diff
@@ -0,0 +1,187 @@
+.header {
+  position: relative;
+
+  @include onTablet {
+    height: 384px;
+  }
+
+  @include onDesktop {
+    height: 655px;
+  }
+
+  &::after {
+    content: "";
+
+    position: absolute;
+    z-index: -1;
+    top: 0;
+    left: 0;
+    right: 0;
+
+    background-image: url(../images/photos/header.png);
+    background-repeat: no-repeat;
+    background-position: center;
+    background-size: cover;
+    height: 300px;
```

Comment:
The height of the background image should be responsive. Consider using percentage or viewport units for the height.